docs/app/utils.js

```diff
@@ -37,9 +46,9 @@ export const scrollToAnchor = () => {
   if (elementTop === 0) return
 
   // hit max scroll boundaries, stop
-  const isScrolledToTop = scrollY === 0
-  const isScrolledToBottom = scrollY + document.body.clientHeight === document.body.scrollHeight
-  const scrollStep = Math.ceil((Math.abs(elementTop / 8))) * Math.sign(elementTop)
+  const isScrolledToTop = offsetY === 0
+  const isScrolledToBottom = offsetY + document.body.clientHeight === document.body.scrollHeight
+  const scrollStep = Math.ceil((Math.abs(elementTop / 8))) * mathSign(elementTop)
```

Comment:
`Math.sign` is [not supported](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Math/sign) in IE at all.